### PR TITLE
Initial Release

### PR DESCRIPTION
**Pull request by TheSpad**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  echo "*** Cleaning Up ***" && \
+  rm /tmp/adguardhomesync.tar.gz
+
+COPY /root /
+
+EXPOSE 8080
+
+VOLUME /config
+
+ENV \
```
Comment (Roxedus) [inline Dockerfile:37]: Move this to the env declaration above?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  fi && \
+  echo "*** Installing AdGuardHome Sync ***" && \
+  curl -o \
+	  /tmp/adguardhomesync.tar.gz -L \
+    "https://github.com/bakito/adguardhome-sync/releases/download/${ADGUARDHOMESYNC_RELEASE}/adguardhome-sync_${ADGUARDHOMESYNC_RELEASE#v}_linux_x86_64.tar.gz" && \
+  tar xzf \
+	  /tmp/adguardhomesync.tar.gz -C \
+	  /app/adguardhome-sync/ && \
+  echo "*** Cleaning Up ***" && \
+  rm /tmp/adguardhomesync.tar.gz
```
Comment (Roxedus) [inline Dockerfile:29]: Internet said; tabs vs spaces?
Spad replies: why not both.

File: Dockerfile (hunk 1 above)
Comment (TheSpad) [reply Dockerfile:37]: Yeah that's a legacy of when I was still building the Go code from source, I'll fix it up

File: Dockerfile (hunk 2 above)
Comment (TheSpad) [reply Dockerfile:29]: Spad is very confused because that's 2 and 4 spaces in vscode

Comment (TheSpad) [reply Dockerfile:29]: Oh, it's because I copied iX's code from overseerr for those lines so not my fault. Fixing now.

**Review by aptalca — APPROVED** [review]

**Pull request merged**